uploadNeuroDB/minc_deletion.pl

```diff
@@ -336,6 +336,15 @@ sub selORdel {
 
 }
 
+# If using SeriesUID, get number of matching files before deleting
+if ($field eq "SeriesUID") {
+    $query = "SELECT COUNT(*) FROM files WHERE SeriesUID = ?";
+    $sth = $dbh->prepare($query);
+    $sth->execute($seriesuid);
+    $sUIDFiles = $sth->fetchrow_array;
+    print "\n ". $sUIDFiles ." files matched with SeriesUID " . $seriesuid . "\n";
```

Comment:
A minor detail: can you remove the space before the filecount?
Makes the messages on the screen more consistent!